### ISSUE TITLE
[FEATURE_REQUEST] Regenerate {{user}}'s message more easily.

### ISSUE THREAD

**Issue by Vibecoder9000** [SillyTavern/SillyTavern#2641]
### Have you searched for similar requests?

Yes

### Is your feature request related to a problem? If so, please describe.

There isn't a particular problem, but the llm handles swapping char card and user description badly and gets confused easily. Not a breaking issue.

### Describe the solution you'd like

The arrowpress/arrow/swipe action only appears on {{char}}'s messages. It'd be great if it were also available for {{user}}. This improves impersonation, as the current method can lose the message in the chat box if you click too soon, has fewer clicks, and is easier to activate. 

### Describe alternatives you've considered

No alternative solutions. A workaround is swapping char's information and putting it in user, but that loses everything but the description and is hard for the model to handle.

### Priority

Low (Nice-to-have)

### Are you willing to test this on staging/unstable branch if this is implemented?

Yes

**Comment by Vibecoder9000** [comment]
Bump, if only for the conveinment user regenerate. Created a ahk script to do this, but it's also annoying because I use pretty much all my keyboard keys.

**Comment by Vibecoder9000** [comment]
Still relevant. Hope it gets added.

**Comment by cloak1505** [comment]
I assume he's asking for swipe button on user message -> Impersonate. If this becomes a thing, then two things: /addswipe functionality and "Swipes for User" option in User Settings.

**Comment by Vibecoder9000** [comment]
Yes. I don't really see a way this can be smoothly added in the same manner as {{char}}'s swipe. The goal is to be able to go back and forth between the user's messages and remove the menu barrier from regeneration.

**Comment by Wolfsblvt** [comment]
User messages don't have swipe functionality. Not sure it makes a lot of sense. Though sure it could be possible.
Sounds like a nice extension idea. Enable user swipes, with impersonation support.

**Comment by Vibecoder9000** [comment]
Well intentioned but hard to implement.

**Comment by Wolfsblvt** [comment]
It's actually not impossible, but not something that I'd add to core.
Would be n interesting idea for an extension for sure, if someone tackles it.

But there is gonna be side effects on how swiping functionality display the arrows in core.
